pkg/ccl/backupccl/restore_planning.go

```diff
@@ -1795,6 +1795,11 @@ func doRestorePlan(
 		if restoreStmt.DescriptorCoverage == tree.AllDescriptors {
 			telemetry.Count("restore.full-cluster")
 		}
+		if restoreStmt.Subdir == nil {
+			telemetry.Count("restore.deprecated-syntax")
```

Comment:
should this be something like `deprecated-subdir-syntax` to give it more description?

Reply:
Done.